app/routes/reminders.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return ['bekah needs to get more banana chips', 'josh needs to do a cartwheel', 'Bekah needs to get out of her funk', 'We both need to listen to some funky music', 'make money']
```

Comment:
1) Let's get bekah out of her funk AND more cartwheels would be good. 2. Let's use Mirage to grab these models, we don't need to hardcode them. Check out [docs for how to findAll records from a store](https://guides.emberjs.com/v2.11.0/models/finding-records/#toc_retrieving-multiple-records) -- this call will trigger Mirage to hand you whatever it's created instead.